react/RatingSummary.tsx

```diff
@@ -11,7 +11,12 @@ import { NormalizedCacheObject } from 'apollo-cache-inmemory'
 import { withApollo } from 'react-apollo'
 import { ProductContext } from 'vtex.product-context'
 import { useCssHandles } from 'vtex.css-handles'
+import { FormattedMessage } from 'react-intl'
+import { Link, canUseDOM } from 'vtex.render-runtime'
+import path from 'ramda/es/path'
```

Comment:
Ramda is a global package in IO, you can use this format:

```suggestion
import { path } from 'ramda'
```